addons/better-terrain/editor/Dock.gd

```diff
@@ -510,3 +593,47 @@ func canvas_input(event: InputEvent) -> bool:
 func canvas_mouse_exit() -> void:
 	draw_overlay = false
 	update_overlay.emit()
+
+
+func _shortcut_input(event) -> void:
+#	print_debug(event)
+	if event is InputEventKey:
+		if event.keycode == KEY_C and event.ctrl_pressed and not event.echo:
+			get_viewport().set_input_as_handled()
+			tile_view.copy_selection()
+		if event.keycode == KEY_V and event.ctrl_pressed and not event.echo:
+			get_viewport().set_input_as_handled()
+			tile_view.paste_selection()
+
+
+##bersenham alg ported from Geometry2D::bresenham_line()
```

Comment:
This is a bit of an odd one, but Godot 4 gets really weird about formatting of ## comments. If you have even one that it doesn't like, it can prevent Godot from parsing scripts for custom comments, and it doesn't tell you why. I believe it requires a space after the double hash. Also, minor typo in algorithm name (should be bresenham).

Reply:
whoops good catch